oaknational/Oak-Web-Application
# Feat/button states

## Description

- Adds button focus/hover states
- It currently doesn't really differentiate between variants "brush" and "minimal", as the figma designs too, but i figure that can be an enhancement in a different PR

## Issue(s)

Progresses #434 

## How to test

1. Go to storybook
2. Hover/focus and buttons
3. You should see states as below

## Screenshots

When focusing:

![image](https://user-images.githubusercontent.com/12934669/183715524-ab72affb-31d4-4ad1-9d2b-f4bd3b586557.png)

When hovering:

![image](https://user-images.githubusercontent.com/12934669/183715638-30b89da6-cbbb-4f0e-9f63-f555f345c750.png)


## Checklist
